client/src/components/Inputs.js

```diff
@@ -14,10 +14,9 @@ const OutlinedSelect = (props) => {
 
     return (
         <React.Fragment>
-            <InputLabel ref={inputLabel} htmlFor={props.selectId} {...props.inputProps}>{props.labelText}</InputLabel>
+            <InputLabel ref={inputLabel} htmlFor={props.selectId} {...props.inputProps}><Typography variant="h4">{props.labelText}</Typography></InputLabel>
             <Select
                 native
-                value={props.value}
```

Comment:
Why did you remove this, in React arn't inputs controlled by state...?

Reply:
I have no idea why this was removed either. It's back now